.travis.yml

```diff
@@ -264,15 +264,21 @@ script:
   echo
 
 - |
+  git remote set-branches --add origin develop
+  git fetch
+  STAGED_FILES_CMD=`git diff --name-only --diff-filter=ACMR origin/develop | grep \\\\.php`
+  FILES=${FILES:-$STAGED_FILES_CMD}
```

Comment:
How does the $FILES var is filled ? What happen on large commit with 1000 files ?

Reply:
@eldy I am going to make a test with a lot of files